app/views/donate/show.html.slim

```diff
@@ -0,0 +1,16 @@
+<script type="text/javascript" id="widget-wfp-script" src="https://secure.wayforpay.com/server/pay-widget.js?ref=button"></script>
+<script type="text/javascript">function runWfpWdgt() {var wayforpay = new Wayforpay(); wayforpay.invoice('https://secure.wayforpay.com/button/b644556e003e4');}</script>
```

Comment:
I'd move click logic from `onclick` parameter to `donate_form.js`, and remove that `_btn.slim` partial.

Reply:
Will try do cleanup